pages/index.js

```diff
@@ -208,6 +225,15 @@ export default function Home() {
             border-radius: 10px;
             padding: 10px;
             font-size: 1.5rem;
+            text-align: center;
+        }
+
+        .button-secondary {
+          color: rgba(33, 33, 33, 1);
+          background-color: rgba(77,208,225, 1);
+          border-radius: 10px;
+          padding: 10px;
+          font-size: 1.5rem;
```

Comment:
`.button-secondary` is missing a `text-align: center;` (which `button-primary` has)